packages/types/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "@formbricks/tsconfig/base.json",
-  "include": ["."],
+  "include": [".", "../ee/multiLanguage/types/*"],
```

Comment:
I'm not sure if we are able from a licensing point of view to import code from `ee` into the surveys or js package. We shouldn't do this if we can avoid this. If they are general types and general lib functions please move them to the `types` and `lib` packages.